texture_manager.c

```diff
@@ -229,20 +189,6 @@ void texture_manager_get_sheet_size(char *url, int *width, int *height) {
         }
     }
 
-    if (fontsheet_map_root) {
```

Comment:
This was deprecated font sheet logic that's not longer supported in devkit and caused lots of errors to be emitted for no reason.